apps/ios/Shared/Views/Chat/ChatView.swift

```diff
@@ -716,31 +717,53 @@ struct ChatView: View {
 
         var revealed: Bool { chatItem == revealedChatItem }
 
+        struct TimeSeparation {
```

Comment:
why is it a struct?

Reply:
we are passing two parameters, any encapsulation beyond maybe ad hoc tuple, is not necessary here. Or just two parameters